SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,90 @@
+This document is a declaration of software quality for the `spdlog` external dependency imported by the `spdlog_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `spdlog` Quality Declaration
+
+TODO: Declare the quality level for `spdlog`
+
+## Version Policy
+
+### Version Scheme
+
+There is a three-part version attached to `spdlog` releases.
+The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
+There is an old ticket where the project's primary developer discusses versioning, but there is no documentation officially stating what approach the developer is taking.
+
+### API/ABI Stability Within a Released ROS Distribution
```

Comment:
Separate out into separate sections.

Reply:
I mean sure, but this doesn't make a lot of sense for an external package - especially one that doesn't have an explicit version policy.